README.md

```diff
@@ -56,6 +56,11 @@ rebuilt and installed.
 
 ## Usage ##
 
+### Enable GPIOs: ###
+Enabling GPIOs is done in sortbootorder by toggling corresponding UARTs. Please refer to:
+
+* [Enabling GPIOs in sortbootorder](https://github.com/pcengines/sortbootorder/blob/52109c940838da4be1e76054d1f325f79d16ac72/README.md)
```

Comment:
@mmkow I believe this should rather point to master. Once we merge that, it will point to the correct file.

Reply:
corrected https://github.com/pcengines/linux-gpio-nct5104d/pull/5/commits/6f8a6ed2323a7b8e4e8d457bc9d328762ac5c354